CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # How to become a contributor and submit your own code
 
-**Table of contents**
```

Comment:
What's up with the changes here?  This is all driven from a central template in synthtool, and would be stomped the following day:
https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/CONTRIBUTING.md